ui.apps/src/main/content/jcr_root/apps/testaem/components/customheader/.content.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+          jcr:primaryType="cq:Component"
+          jcr:title="Custom header component"
+          componentGroup="My AEM test project - Contentt"/>
```

Comment:
pls fix typo in "Contentt" in all files

Reply:
New component group "Custom components (News)" has been created instead of "My AEM test project - Contentt".

corrected in the last commit.